doc/nrf/doc_styleguide.rst

```diff
@@ -131,23 +152,31 @@ These are the |gl| for the doxygen-based API documentation.
 General documentation guidelines
 ================================
 
-#. Always use full sentences, except for descriptions for variables, structs, and enums, where sentence fragments with no verb are accepted, and always end everything with period.
-#. Everything that is documented must belong to a group (see below).
-#. Use capitalization sparingly. When in doubt, use lowercase.
-#. Line breaks: In doxygen, break after 80 characters (following the dev guidelines). In RST, break after each sentence.
-#. ``@note`` and ``@warning`` should only be used in the details section, and only when really needed for emphasis.
-   Use notes for emphasis, and warnings only if things will really *really* go wrong if you ignore the warning.
+* Always write full sentences, and end them with a period.
+
+  * *Exception:* Sentence fragments are acceptable for descriptions of variables, structs, and enums.
+* Ensure that all documented items belong to a correct group (see the section below).
+* Use capitalization sparingly.
+  When in doubt, use lowercase.
+* Follow the line breaking rules:
+
+  * In doxygen, break the line after 80 characters (following the dev guidelines).
+  * In RST, break the line after each sentence.
```

Comment:
As we are in the Doxygen section, shoudn't this line be moved to RST part? Is this information relevant for Doxygen users?

Reply:
Very good point. Will move this out.